CHANGELOG

```diff
@@ -2,6 +2,13 @@
 Changelog
 *********
 
+0.45.0 (unreleased)
```

Comment:
There seem to be a bunch of changes that are unrelated to this PR. @samchrisinger Can you look into this?